lua/weapons/arccw_base/sh_firing.lua

```diff
@@ -705,7 +705,7 @@ function SWEP:DoRecoil()
 
         rmul = rmul * b
         recs = recs * b
-        recv = recv / b
+        recv = recv * b
```

Comment:
this was having the opposite effect as was intended
but probably wasn't noticed due to most used values being very small